Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -45,27 +45,22 @@ HexEditorWidget::HexEditorWidget()
     m_search_results_container = *find_descendant_of_type_named<GUI::Widget>("search_results_container");
 
     auto update_statusbar_selection_and_offset_info = [this](int selection_start, int selection_end) {
-        if (m_show_selection_information_as_hex)
-        {
+        if (m_show_selection_information_as_hex) {
             m_statusbar->set_text(2, String::formatted("Selection Start: {:#x}", selection_start));
             m_statusbar->set_text(3, String::formatted("Selection End: {:#x}", selection_end));
-            m_statusbar->set_text(4, String::formatted("Selected Bytes: {:#x}", m_editor->selection_size()));
-        }
-        else
-        {
+        } else {
             m_statusbar->set_text(2, String::formatted("Selection Start: {}", selection_start));
             m_statusbar->set_text(3, String::formatted("Selection End: {}", selection_end));
-            m_statusbar->set_text(4, String::formatted("Selected Bytes: {}", m_editor->selection_size()));
         }
     };
 
     m_editor->on_status_change = [this, update_statusbar_selection_and_offset_info](int position, HexEditor::EditMode edit_mode, int selection_start, int selection_end) {
+        m_statusbar->set_text(4, String::formatted("Selected Bytes: {}d ({:#x})", m_editor->selection_size(), m_editor->selection_size()));
```

Comment:
For the second number, it's not really clear that that's hex, because the start and end number may show decimal (without a trailing "d"). I'd suggest switching hex and decimal marking because that's how the English language works, but it's up to you to find a good solution.

Reply:
I still think that the unicode decimal symbol is very clear, and for programmers a prefix of `0x` will also always be clear.

Reply:
Then we should have 0x :)

Reply:
0x at the beginning of the number is printed by `{:#x}` formatting already. Also, I don't really know how to add that symbol someone suggested before to Serenity fonts, but 'd' suffix seems to be clear for me as it is one of two most common math notations for decimals IMHO. But maybe in the future someone would change this.